global.json

```diff
@@ -1,6 +1,5 @@
 {
-  "projects": [ "src", "tests" ],

```

Comment:
Based on [this](https://docs.microsoft.com/en-us/dotnet/core/tools/global-json), the `projects` property no longer exists.